ports/freecol/port.json

```
@@ -0,0 +1,34 @@
+{
+  "version": 4,
+  "name": "freecol.zip",
+  "items": [
+    "FreeCol",
```

Ah finally I found something ! 😅

```suggestion
    "FreeCol.sh",
```